### PR TITLE
added the Raccoon background

### PR DESCRIPTION
**Pull request by VanessaMedinaUNR**

### CODE REVIEW

File: theme_items/raccoon scene.png
Comment (dinaolmelak) [inline theme_items/raccoon scene.png:1]: Can you rename it to `raccoon_scene` ?

**Review by dinaolmelak — CHANGES_REQUESTED** [review]
Looks good to me 👍🏽, I see the couch is also fixed, cool 👍

**Comment by dinaolmelak** [comment]
@VanessaMedinaUNR I meant like the `raccoon scene.png`

File: theme_items/raccoon scene.png
Comment (VanessaMedinaUNR) [reply theme_items/raccoon scene.png:1]: just did it!

Comment (dinaolmelak) [reply theme_items/raccoon scene.png:1]: Awesome @VanessaMedinaUNR  😄  I see you fixed it!!

**Review by dinaolmelak — APPROVED** [review]
LGTM!

**Review by dinaolmelak — CHANGES_REQUESTED** [review]
Make sure it is up to date with the `main`

**Comment by dinaolmelak** [comment]
Awesome 😄

**Review by dinaolmelak — APPROVED** [review]

**Pull request merged**